smartsim/database/orchestrator.py

```diff
@@ -165,6 +165,43 @@ def check_cluster_status(self):  # cov-wlm
         if trials == 0:
             raise SmartSimError("Cluster setup could not be verified")
 
+    def get_address(self):
+        """Return database addresses
+
+        :return: addresses
+        :rtype: list[str]
+        """
+        if not self._hosts:
+            raise SmartSimError("Could not find database address")
+        if len(self._hosts) > 1:
+            self.check_cluster_status()
+        elif not self.is_active():
+            raise SmartSimError("Database is not active")
+        addresses = []
+        for host, port in itertools.product(self._hosts, self.ports):
+            addresses.append(":".join((host, port)))
+        return addresses
+
+    def is_active(self):
+        """Check if database is running
+
+        :returns: True if database is active, False otherwise
+        :rtype: bool
+        """
+        try:
+            host = self._hosts[0]
```

Comment:
what happens if this is called on an ``Orchestrator`` object that hasn't been launched yet?

Reply:
wouldn't `if not self._hosts: raise SmartSimError(...)` in `get_address()` account for that before `is_active()` is even called?

Reply:
Since this method is user facing, the user should also be able to call this directly. 

Ex. 

'''Python
db = Orchestrator()
db.is_active()
'''

Reply:
gotcha. so if I add the same check in `is_active()` that should account for this, right?